CODEOWNERS

```diff
@@ -1,18 +1,41 @@
 # Code ownership information.
-# See https://help.github.com/articles/about-code-owners/ for details.
+# See
+# https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners
+# for details.
 
+# Order is important; the *last* matching pattern takes precedence.
+
+# Default
+#
 # These owners will be the default owners for everything in the repo. Unless a
-# later match takes precedence, # @trusted-contributors will be requested for
-# review when someone opens a pull request.
+# later match takes precedence, @collectd/trusted-contributors will be
+# requested for review when someone opens a pull request.
+*       @collectd/trusted-contributors
 
-/src/intel_pmu.c	@kwiatrox @sunkuranganath
-/src/intel_rdt.c	@kwiatrox @sunkuranganath
-/src/ipmi.c		@anaudx @rjablonx
-/src/mcelog.c		@kwiatrox @sunkuranganath
-/src/virt.c		@anaudx @rjablonx
+# Per-plugin owners
+#
+# These plugins are owned by subject matter experts and require their review.
+/src/intel_pmu.c	@collectd/intel
+/src/intel_rdt.c	@collectd/intel
+/src/ipmi.c		@collectd/intel
+/src/mcelog.c		@collectd/intel
+/src/virt.c		@collectd/intel
 # TODO(#2926): Add the following owners:
-#/src/redfish.c		@kkepka @mkobyli
+#/src/redfish.c		@collectd/intel
+
+# Core
+#
+# The daemon and some plugins with a huge "blast radius" are considered "core"
+# to the collectd project and require review form a "core owner".
+/src/daemon/		@collectd/core-maintainers
+/src/liboconfig/	@collectd/core-maintainers
+/src/utils/ignorelist/	@collectd/core-maintainers
+/src/utils/metadata/	@collectd/core-maintainers
+/src/utils/mount/	@collectd/core-maintainers
+/src/cpu.c		@collectd/core-maintainers
```

Comment:
We've had exec and interface in the preliminary list of core plugins as well.

Reply:
Done.

Reply:
Did you really add these? I can not find them in the latest commit?

Reply:
I did indeed forget to force-push the amended commit. Fixed.